elasticpath/react-pwa-reference-storefront
# RS-626 A-Frame Implementation

Description:


**NOTE: 
1. Still needs to be tested by being deployed behind https security DONE
2. Tested with Oculus Quest DONE
4. and Iphone devices DONE
5. Test items with both VR and AR assets DONE
6. Test with Android device DONE
7. Ensure storybook is working and documented DONE
8. Error on Iphone when exitting screen.  Image is not fullsized. DONE
9. The exit button disappears when going fullscreen mode.  DONE
10.  Change the loading screen on IOS. DONE

Linting:


Component Updates:


Tests:


Documentation:
